### PR TITLE
fix(dynamic-sampling): Add query timeout logging for more functions

### PR DESCRIPTION
**Pull request by iambriccardo**
This PR adds more timeouts loggers and increases the time for execution of org recalibration.

### CODE REVIEW

File: src/sentry/dynamic_sampling/tasks/boost_low_volume_transactions.py [hunk 1]
```diff
@@ -292,6 +292,8 @@ def get_orgs_with_project_counts(
             break
     if len(last_result) > 0:
         yield [org_id for org_id, _ in last_result]
+    else:
+        log_query_timeout(query="get_orgs_with_project_counts", offset=offset)
```
Comment (RaduW) [inline src/sentry/dynamic_sampling/tasks/boost_low_volume_transactions.py:296]: Wrong place of the log ( it should be one block higher)

File: src/sentry/dynamic_sampling/tasks/common.py [hunk 2]
```diff
@@ -110,6 +110,8 @@ def get_active_orgs_with_projects_counts(
             break
     if len(last_result) > 0:
         yield [org_id for org_id, _ in last_result]
+    else:
+        log_query_timeout(query="get_active_orgs_with_projects_counts", offset=offset)
```
Comment (RaduW) [inline src/sentry/dynamic_sampling/tasks/common.py:114]: also wrong place :-) ( one block up)

**Review by RaduW — CHANGES_REQUESTED** [review]
Need to move the:
```
else: 
    log_query_timeout(...)
```

blocks to their proper place

File: src/sentry/dynamic_sampling/tasks/common.py (hunk 2 above)
Comment (iambriccardo) [reply src/sentry/dynamic_sampling/tasks/common.py:114]: Talking about why brackets are better than tabs 🤣

**Review by RaduW — APPROVED** [review]
Good to go

**Pull request merged**